src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcInternalHead.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcInternalGtmScript
+@import uk.gov.hmrc.hmrcfrontend.config.AssetsConfig
+
+@this(hmrcInternalGtmScript: HmrcInternalGtmScript, assetsConfig: AssetsConfig)
+
+@(headBlock: Option[Html] = None, nonce: Option[String] = None)(implicit request: RequestHeader, messages: Messages)
+@hmrcInternalGtmScript(nonce)
+<!--[if lte IE 8]>
+<script src="@assetsConfig.html5ShivJsUrl"></script>
+<link href="@assetsConfig.hmrcFrontendIe8CssUrl" media="all" rel="stylesheet" type="text/css" />
+<![endif]-->
+<!--[if gt IE 8]><!-->
+<link href="@assetsConfig.hmrcFrontendCssUrl" media="all" rel="stylesheet" type="text/css"@nonce.map {n=> nonce="@n"} />
```

Comment:
@JoPintoPaul looks good to me, and if no csp changes are needed, only thought I have is should we also include in the guidance a note about not using this alongside HmrcHead, that it's an alternative for internal services / maybe that if you are using HmrcLayout to use HmrcInternalGtmScript directly or make sure to swap HmrcHead for HmrcInternalHead. Not sure it would break anything but they would be including stylesheets twice